topdown/bits_test.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2019 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+package topdown
+
+import "testing"
+
+func TestBuiltinBitsOr(t *testing.T) {
+	tests := []struct {
+		note     string
+		rules    []string
+		expected interface{}
+	}{
+		{"basic bitwise-or", []string{`p[x] { x := bits.or(7, 9) }`}, `[15]`},
+		{"or with zero is value", []string{`p[x] { x := bits.or(50, 0) }`}, `[50]`},
```

Comment:
Please add some negative tests for the builtins with invalid types passed in (both for the function signature check like passing in a string or something and then for passing in a valid `ast.Number` but a float that fails to convert to an integer.

Reply:
Done!